### DISCUSSION TITLE
Zerotier获取到IPV4地址但是无法访问

### DISCUSSION THREAD

**Discussion by rawvoid** [immortalwrt/immortalwrt#1025] [Q&A] [answered]
### Describe the bug

无法通过Zerotier的IP地址访问路由器管理页面，页面显示`ERR_CONNECTION_REFUSED`；`ping`显示`无法连接到端口`。只有在路由器本机通过`curl 10.0.0.161`可以访问。

Zerotier 连接状态：

![image](https://github.com/immortalwrt/immortalwrt/assets/70388114/42f13a8c-8304-4dce-ba5c-7ff5ed89ca14)
![image](https://github.com/immortalwrt/immortalwrt/assets/70388114/65d535d1-c8af-4fe9-ac85-7d6cd1a7cf67)

Zerotier局域网其他设备访问情况：

![image](https://github.com/immortalwrt/immortalwrt/assets/70388114/45991874-cd79-4668-ae7b-5accaea47c3a)
![image](https://github.com/immortalwrt/immortalwrt/assets/70388114/70276c6c-0116-4e67-b87f-9cab23bc2ef8)

路由器本机通过curl可以访问

![image](https://github.com/immortalwrt/immortalwrt/assets/70388114/fec2d62f-67b7-482d-b07a-1b3c4c8e1761)

### ImmortalWrt version

r26809-8fb22b51b6

### ImmortalWrt target/subtarget

x86/64

### Device

VMware, Inc. VMware Virtual Platform

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by Saxon-Sun** [comment, marked as answer]
防火墙—常规设置——入站数据——改成接受

**Reply by skefy** [reply]
在防火墙中增加端口转发是不是一样的效果呢？

**Reply by 1715173329** [reply]
正确配置是在通信规则里面运行 zt 相关 ip 入站，而不是全局接受。

**Comment by rawvoid** [comment]
> 防火墙—常规设置——入站数据——改成接受

感谢您关注这个问题，按照您的设置后现在可以访问了。
